wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/SimpleTextField.swift

```diff
@@ -163,8 +159,14 @@ final class SimpleTextField: UITextField, DynamicTypeCapable {
 
 // MARK: SimpleTextField Extension
 extension SimpleTextField: SimpleTextFieldValidatorDelegate {
-    func textFieldValueChanged(_ value: String?) {
-        textFieldDelegate?.textField(self, valueChanged: value.value ?? .error(.empty))
+
+    func textFieldValueChanged(_ text: String?) {
+        let validator = SimpleTextFieldValidator()
+        let newValue = { () -> SimpleTextField.Value in
+            guard let text else { return .error(.empty) }
+            return if let error = validator.validate(text: text) { .error(error) } else { .valid(text) }
```

Comment:
This formatting would align with line 46:
```suggestion
            if let error = validator.validate(text: text) {
                return .error(error)
            } else { 
                return .valid(text)
            }
```